openedx/features/course_bookmarks/static/course_bookmarks/fixtures/bookmark_button.html

```diff
@@ -1,12 +1,11 @@
-
 <div class="message-banner" aria-live="polite"></div>
 
 <div class="xblock xblock-student_view xblock-student_view-vertical xblock-initialized">
     <div class="bookmark-button-wrapper">
-      <button class="btn bookmark-button"
-        aria-pressed="false"
-        data-bookmark-id="bilbo,usage_1">
-         <span class="bookmark-text">Bookmark this page</span>
+        <button class="btn bookmark-button"
+                aria-pressed="false"
+                data-bookmark-id="bilbo,usage_1">
```

Comment:
We don't actually have an HTML styleguide, but the pattern I prefer for indentation here is:

```html
<button
    class="btn bookmark-button"
    aria-pressed="false"
    data-bookmark-id="bilbo,usage_1"
>
    <children>
</button>
```

That matches the eslint-config-airbnb indentation standard for JSX, which is the closest thing we have for a styleguide for HTML.